lib/xfstests_utils.pm

```diff
@@ -601,12 +601,16 @@ sub test_run_without_heartbeat {
         sleep 2;
         copy_all_log($category, $num, $fstype, $btrfs_dump, $raw_dump, $scratch_dev, $scratch_dev_pool, 1);
 
-        prepare_system_shutdown;
-        reset_consoles if check_var('DESKTOP', 'textmode');
-        ($virtio_console == 1) ? power_action('reboot') : send_key 'alt-sysrq-b';
-        reconnect_mgmt_console if is_pvm;
-        check_var('DESKTOP', 'textmode') ? $self->wait_boot_textmode : $self->wait_boot;
-        is_public_cloud() ? select_console('root-console') : select_serial_terminal();
```

Comment:
Here need to keep `select_serial_terminal;` if you move is_public_cloud condition out.